.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,68 @@
+name: Clang Format Check
+on:
+  pull_request:
+    paths:
+      - 'Sources/**'
+      - 'Tests/**'
```

Comment:
Could you please add `Samples/Common/Sources/CrashTriggers` too, it's an ObjC lib